docs/examples/viz_shapes.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+==============
+Simple Shapes
+==============
+
+This example shows how to use the UI API. We will demonstrate how to draw
+some geometric shapes from FURY UI elements.
+"""
+
+##############################################################################
+# First, a bunch of imports
+
+from fury.ui import Rectangle2D, Disk2D, UIContext
+from fury.window import (
+    Scene,
+    ShowManager,
+)
+
+##############################################################################
+# Using UI v2
+
+UIContext.set_is_v2_ui(True)

```

Comment:
I wasnt very satisfied with this approach (check the usage below to understand the mess), but added it as a quick starter which we can iterate on.

Feel free to suggest other approaches.

Reply:
Not sure of what is the goal of `UIContext`

Reply:
So the `UIContext` works as a Global Singleton to provide context to the UI.

For example
1. UI needs Canvas Size to calculate and reposition itself due to inverted y axis, so instead of passing the size or the renderer to each UI we store it here in the context and the UI fetches it whenever required.
1. To implement the concept of hover, UI focus (currently active UI) we need to have a reference to hot and active UI globally so that's too stored here.